packages/gatsby-image/src/index.js

```diff
@@ -250,24 +258,30 @@ class Image extends React.Component {
       itemProp,
     } = convertProps(this.props)
 
+    const shouldDisplay = this.state.imgLoaded || this.state.fadeIn === false
```

Comment:
not sure but maybe shouldLazyLoad is a better name? not sure if lazyload is the right term here

Reply:
I think the condition for that boolean gives the context/clarity what it is for, it was just more appropriate to summarize it as a `should` variable where it was used so the code for the style was still easy to read when the logic would otherwise get a bit long/messy.

If `shouldDisplay` doesn't feel right, how about `shouldReveal`? The variable is describing when the image is ready to display by toggling the opacity, a `shouldLazyLoad` seems to describe the wrong relationship, you could try `isLazyLoaded` or `isReady`(since ultimately it's a true value you're after, there isn't really a distinction to lazy-loading or not.

Reply:
👍 for shouldReveal! let me change that and merge afterwards!